src/main/java/at/hannibal2/skyhanni/config/features/inventory/AuctionHouseConfig.java

```diff
@@ -49,4 +49,10 @@ public class AuctionHouseConfig {
     )
     @ConfigEditorKeybind(defaultKey = Keyboard.KEY_NONE)
     public int copyUnderbidKeybind = Keyboard.KEY_NONE;
+
+    @Expose
+    @ConfigOption(name = "Price Website", desc = "Adds a button to the Auction House that will open the item page in §ccoflnet.com§7.")
```

Comment:
Why is coflnet red?
Shouldn't it be yellow?
also what Ekwav said. sky.coflnet.com is the website